ui/src/components/photoGallery/ProtectedMedia.js

```diff
@@ -18,17 +21,24 @@ const getProtectedUrl = url => {
 /**
  * An image that needs authorization to load
  */
-export const ProtectedImage = ({ src, ...props }) => (
-  <img
-    key={src}
-    {...props}
-    src={getProtectedUrl(src)}
-    crossOrigin="use-credentials"
-  />
-)
+export const ProtectedImage = ({ src, loading, ...props }) => {
```

Comment:
I like the use of a new prop to specify `eager` and `lazy` loading, but as `loading` is used a lot as a boolean prop I think it would be better to called it something else like `loadingMode`.

Also I think a quick comment explaining that it should be either `eager` or `loading` and that you need to manually update it with `lazyLoad` if set to `loading` would be nice, as it's no obvious directly from the code.

Reply:
I understand your idea and reasoning to suggest renaming of `loading` to `lazyLoad` but... I like the idea to keep component props names as close as possible to the underlying HTML elements. For example, ProtectedImage `src` prop is used to set the `src` attribute of the underlying `<img />` HTML element and `loading` prop is used to set `loading` attribute of `<img />` . 
In my opinion, it is much easier to debug code if the variable names do not change when passed as an argument to the descending functions. 

Anyway, if you really think that setting `lazyLoad` prop to `<ProtectedImage />` it will be moe intuitive approach for setting `loading` attribute of underlying `<img />` element - I am fine with refactoring it, just confirm iit.

Reply:
That makes sense, I see pros and cons for each approach, so feel free to keep it as is if you think that's the best option. If we just make sure to document it then it should be easy to understand nevertheless.

Reply:
Fixed it, check the latest commit :)